### PR TITLE
Feature/priority places polygons layer

### PR DESCRIPTION
**Pull request by weberjavi**
[PIVOTAL TASK
](https://www.pivotaltracker.com/story/show/168250649)[INVISION PROTOTYPE](https://projects.invisionapp.com/d/main#/console/17484384/374973459/preview)

This PR adds the `Priority places for Biodiversity` featured map polygons and taxa selector.
We are getting the biodiversity polygons from a proprietary data table on ArcGIS online and adding the polygons to the map through a `graphic layer`.
Some condition have been added to the `featured places layer` logic to display only selected taxa markers (Markers styles will be handled in a different PR)

### CODE REVIEW

**Comment by weberjavi** [comment]
Added the WIP label as some updates on the design has been made.
https://projects.invisionapp.com/share/9GRXK9GQU6M#/screens/374973459_HE_Map_-_Featured_Default

**Comment by weberjavi** [comment]
Will let you know when is ready again

**Comment by weberjavi** [comment]
@aabdaab , @kfilip94 this PR ☝️  is ready to be reviewed. Be aware that no data is displayed when a place is selected. This will be handled on another PR

**Comment by weberjavi** [comment]
@aabdaab , @kfilip94 adding again the WIP label ☝️ 
Since we are going to release to demo today and probably tomorrow to production it does not makes sense to deploy this feature without data, so instead of doing this task in another PR will be adding code to this open PR.

**Comment by weberjavi** [comment]
@aabdaab , @kfilip94 ready to review ☝️

**Comment by simaob** [comment]
This is a high priority. Don't forget to review it ASAP.

**Comment by aabdaab** [comment]
the images will follow and shouldn't care about the placeholder image?
image

File: src/utils/layer-manager-utils.js [hunk 1]
```diff
@@ -48,6 +48,8 @@ export const layerManagerOrder = (datasets, activeLayers, callback) => {
   callback({ activeLayers: updatedLayers });
 };
 
+export const layerInMap = (layerTitle, map) => map.layers.items.some(l => l.title === layerTitle);
```
Comment (aabdaab) [inline src/utils/layer-manager-utils.js]: I think we can remove that, I haven't seen it used anywhere

Comment (weberjavi) [reply src/utils/layer-manager-utils.js]: I'm working on a local branch to change how we are handling layers and I already have updated this `layer-manager-utils` do you think it makes sense to merge this branch as it is and wait for the `load-improvements` branch to handle this dead code?

**Pull request merged**